geomstats/geometry/full_rank_correlation_matrices.py

```diff
@@ -0,0 +1,143 @@
+"""The manifold of full-rank correlation matrices."""
+
+import math
+
+import geomstats.backend as gs
+from geomstats.geometry.embedded_manifold import EmbeddedManifold
+from geomstats.geometry.general_linear import GeneralLinear
+from geomstats.geometry.matrices import Matrices
+from geomstats.geometry.quotient_metric import QuotientMetric
+from geomstats.geometry.spd_matrices import SPDMatrices, SPDMetricAffine
+from geomstats.geometry.symmetric_matrices import SymmetricMatrices
+
+EPSILON = 1e-6
+TOLERANCE = 1e-12
+
+class FullRankCorrelationMatrices(SPDMatrices, EmbeddedManifold):
```

Comment:
What I had in mind was that this class should inherit FiberBundle with total space SPD, instead of SPD inheriting FIberBundle